src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1524,17 +1524,28 @@ void runtime::cpu::CPU_ExternalFunction::build()
         }
         else
         {
-            for (const auto& p : enables)
+            std::advance(functor, ctx->pc);
```

Comment:
@jbobba can I change `enables` and `functors` to use `std::vector`? We might take a negligible perf hit when those get resized at a *build* step (which could also be avoided if we pass the size in `CPU_ExternalFunction` c-tor) but then we will be able to just use regular `for` loop with an index w/o this iterator voodoo?